src/scenes/MerchStorePage/Redemption/RedeemedPage.tsx

```diff
@@ -0,0 +1,96 @@
+import { useCallback } from 'react';
+import { graphql, useFragment } from 'react-relay';
+import styled from 'styled-components';
+
+import { Button } from '~/components/core/Button/Button';
+import { HStack } from '~/components/core/Spacer/Stack';
+import { BaseM } from '~/components/core/Text/Text';
+import { useModalActions } from '~/contexts/modal/ModalContext';
+import { RedeemedPageFragment$key } from '~/generated/RedeemedPageFragment.graphql';
+import ArrowUpRightIcon from '~/icons/ArrowUpRightIcon';
+
+import { getObjectName } from '../getObjectName';
+import RedeemedItem from './RedeemedItem';
+
+type Props = {
+  merchTokenRefs: RedeemedPageFragment$key;
+};
+
+export default function RedeemedPage({ merchTokenRefs }: Props) {
```

Comment:
supernit: there's a little layout shift when i switch between the tabs in this modal. ideally, i would like the height of the modal to remain constant as the user moves between each tab. could we add a `min-height` to the modal content?

https://user-images.githubusercontent.com/12162433/208024347-f06c8e61-b3ef-4d06-af33-1f29d2e12adf.mov

Reply:
Got it! I've set the minimum modal height to `250px`


https://user-images.githubusercontent.com/4480258/208027983-7e76fb2d-af10-40da-b391-76ef4b673d7d.mp4